flocker/testtools/__init__.py

```diff
@@ -881,14 +953,23 @@ def run_process(command, *args, **kwargs):
     """
     kwargs["stdout"] = PIPE
     kwargs["stderr"] = STDOUT
-    process = Popen(command, *args, **kwargs)
-    output = process.stdout.read()
-    status = process.wait()
-    result = _ProcessResult(command=command, output=output, status=status)
-    if result.status:
-        raise _CalledProcessError(
-            returncode=status, cmd=command, output=output,
-        )
+    action = start_action(
+        action_type="run_process", command=command, args=args, kwargs=kwargs)
+    with action:
+        process = Popen(command, *args, **kwargs)
+        output = process.stdout.read()
+        status = process.wait()
+        result = _ProcessResult(command=command, output=output, status=status)
+        # TODO: We should be using a specific logging type for this.
+        Message.new(
+            command=result.command,
```

Comment:
Perhaps at least add a unique message_type to help find these messages :question: